libs/methodologies/bold/rule-processors/mass-id/src/time-interval-check/time-interval-check.processor.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { loadParentDocument } from '@carrot-fndn/shared/methodologies/bold/io-helpers';
+import {
+  stubDocument,
+  stubDocumentEvent,
+} from '@carrot-fndn/shared/methodologies/bold/testing';
+import { DocumentEventName } from '@carrot-fndn/shared/methodologies/bold/types';
+import {
+  type RuleInput,
+  type RuleOutput,
+  RuleOutputStatus,
+} from '@carrot-fndn/shared/rule/types';
+import { random } from 'typia';
+
+import { TimeIntervalCheckProcessor } from './time-interval-check.processor';
+
+jest.mock('@carrot-fndn/shared/methodologies/bold/io-helpers');
+
+describe('TimeIntervalCheckProcessor', () => {
+  const ruleDataProcessor = new TimeIntervalCheckProcessor();
+  const documentLoaderService = jest.mocked(loadParentDocument);
+
+  it.each([
+    {
+      dropOffEventDate: '2024-02-25T12:00:00.000Z',
+      recycledEventDate: '2024-02-27T11:00:00.000Z',
+      resultComment: ruleDataProcessor['RESULT_COMMENT'].REJECTED,
+      resultStatus: RuleOutputStatus.REJECTED,
+      scenario: 'Time interval is less than 60 days (2 days)',
+    },
```

Comment:
Would it be the case to export and reuse these test scenarios for both unit and e2e tests?